test/parallel/test-event-emitter-set-max-listeners-side-effects.js

```diff
@@ -26,7 +26,6 @@ const events = require('events');
 
 const e = new events.EventEmitter();
 
-assert(!(e._events instanceof Object));
```

Comment:
Removed this as in fast path we don't have `__proto__: null` on the events and therefor it is an object